moto/ec2/exceptions.py

```diff
@@ -481,6 +481,14 @@ def __init__(self, vpc_id, max_cidr_limit):
         )
 
 
+class UnsupportedTenancy(EC2ClientError):
+    def __init__(self, tenancy):
+        super(UnsupportedTenancy, self).__init__(
+            "(UnsupportedTenancy",
```

Comment:
nitpick: I'm assuming the error type should be changed to `"UnsupportedTenancy"` (without the leading bracket `(`).